src/Arachne.cc

```diff
@@ -544,8 +506,10 @@ yield() {
     if (!core.loadedContext)
         return;
     if (core.localOccupiedAndCount->load().numOccupied == 1 && !shutdown &&
-        !core.coreReadyForReturnToArbiter)
+        !core.coreReadyForReturnToArbiter) {
+        checkForArbiterRequest();
```

Comment:
Probably add a comment to explain why we put `checkForArbiterRequest()` here? e.g., only check Arbiter request when it is the last thread on this core.

Reply:
Good point. Updated.